DfE.FindInformationAcademiesTrusts/assets/sass/_components.scss

```diff
@@ -1,4 +1,6 @@
 @import "./variables";
+@import "govuk-frontend/govuk/all";
+@import "components/app-search";
```

Comment:
can we name the file `search.scss` and just add the `app-` prefix on the class names?

Reply:
Sure :-)